source/torso_calibration.rst

```diff
@@ -0,0 +1,91 @@
+If the sensor are not properly calibrated, the calculation of the initial torso position
+could be incorrect.
+This should be easy to detect since the reported torso position will always be off by
+more than +/-10cm.  This problem is more likely to occur if the torso is powered on
+in the "up" position than the "down" position.
+
+After shipping the robot, it may be a good idea to verify torso calibration.
+
+Torso Calibration Tool
+-------------------------
+In newer releases of the fetch_drivers package here is a tool
+to calibrate or verify the torso sensor measurement
+The tool has two options: calibrate and verify.
+The calibrate option will calibrate the sensors and update parameters
+stored on the torso controller.  The verfiy option will only verify that the
+torso calbration is good, it will not change any stored parameters.
+
+For both options the torso will travel through it entire range of
+motion while sensor data is collected.  While the tool is being run,
+the robot driver will be stopped and the robot arm will not hold its position.
+Because of this, the arm should be tucked or soft fabric or cardboard
+should be placed between arm and base to avoid scratching any covers.
+
+
+Torso Calibration Procedure
+---------------------------
+.. warning::
+    Durning torso calibration the arm will not hold position.
+    Place cardboard or soft fabric between arm and base to avoid
+    scratching covers during data collection.
+
+Follow these steps in order to verify or calibrate the torso position
+sensors:
+
+1. Move torso to lowest position, and tuck the arm.
+
+2. Place a protective barier between the arm and top base cover.
+
+3. Disable robot drivers by running : ``sudo service robot stop``
+
+4. Run torso calibration tool:
+
+  * To run calibration : ``rosrun fetch_drivers torso_calibrate calibrate``
+  * To verify calibration : ``rosrun fetch_drivers torso_calibrate verify``
+
+6. Wait for torso to collect sensor data.  The torso will move upwards in small increments through the entire range of motion.  A clicking sound will be produced by the torso while moving, and is normal.
+
+7. Cycle Run-stop (optional).  Sometimes tool will request that run-stop be cycled
+   after it completes.  Cycling run-stop will cycle power to the torso controller board,
+   and is required in some situations.
+
+8. On tool has completed, restart robot driver with ``sudo service robot start``
+
+Verify Output
+-------------
+Verification will produce output stating whether sesnors are well calibrated.
+If the sensor calibration is good the tool will output something similar to:
+
+  VERIFY PASSED : max sensor error of 0.0109411 is within acceptable limit
+
+Otherwise it will produce output like:
+
+  VERIFY FAILED : max sensor error of 0.0501323 is larger than acceptable limit of 0.04
+
+When verification fails, run calibration produce.
+
+.. note::
+  The value for max sensor error is the mismatch between the two torso sensors.
```

Comment:
The value in the print statement is actually in meters.   I think if we get the logpro integration to work, I might remove the value all together, because it is misleading, and no longer needed.